index.js

```diff
@@ -9,6 +9,12 @@ const memo = async(key, fn, ttl, forceUpdate) => {
     }
   }
   const result = await fn();
+  // always set the header showing when it was cached:
+  if (!result.headers) {
+    result.headers = { 'x-cache': new Date().getTime() };
```

Comment:
x-arc-cache please

Reply:
there's dupe code here.  can you just do 
```
if (!results.headers) {
  results.headers = {}
}
results.headers[...]
```

Reply:
That will break if results is not an object.

Reply:
For example if result is a number, adding result.headers to it does nothing and the next line will throw ' Cannot set property 'x-arc-cache' of undefined' when you try to set the header.

Reply:
how is that different than what you have?